x/tokenfactory/client/cli/query.go

```diff
@@ -122,7 +122,7 @@ func GetCmdDenomsFromCreator() *cobra.Command {
 }
 
 // GetCmdDenomAuthorityMetadata returns the authority metadata for a queried denom
-func GetCmdDenomBeforeSendHook() *cobra.Command {
+func GetCmdDenomBeforeSendListner() *cobra.Command {
```

Comment:
Same question as before, are we intentionally keeping the below "denom-before-send-hook" nomenclature despite changing it to "before send listener"?

Reply:
Ah good catch! Fixed!